src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -371,6 +379,11 @@ namespace AppInstaller::CLI::Workflow
 
         table.Complete();
 
+        if (table.IsEmpty())
+        {
+            context.Reporter.Info() << Resource::String::NoInstalledPackageFound << std::endl;
+        }

```

Comment:
Are there cases of this not outputting a message?  It currently does so when nothing is found.

Reply:
We output "no package found" during EnsureMatchesFound workflow task. But in this method, when filter to update only is true, we further filter the search result, it's possible after filtering the table is empty. and the table outputs 2 empty lines. So I made the change to the table and output no package found for this case

---
In reply to: [518338774](https://github.com/microsoft/winget-cli/pull/634#discussion_r518338774) [](ancestors = 518338774)